<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LTPCHE102076955" name="com.pom.crimson.testcases.explore.smoke.HomePageTest" tests="11" failures="1" timestamp="2021-01-03T08:46:26 IST" time="1002.692" errors="2">
  <testcase name="verifyNavigationToExplorePage_HomePageTest" time="37.195" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyNavigationToMomentsLogPage_HomePageTest" time="34.045" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyNavigationToRecommendedContentAndClickSeeAll_HomePageTest" time="49.701" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyNavigationToContentPage_HomePageTest" time="94.857" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyLike_HomePageTest" time="115.188" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyUnlike_HomePageTest" time="146.828" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.pom.crimson.listeners.LogAssert.onAssertFailure(LogAssert.java:41)
at org.testng.asserts.SoftAssert.doAssert(SoftAssert.java:23)
at org.testng.asserts.Assertion.fail(Assertion.java:132)
at com.pom.crimson.testcases.explore.smoke.HomePageTest.verifyUnlike_HomePageTest(HomePageTest.java:191)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyUnlike_HomePageTest -->
  <system-out/>
  <testcase name="verifyBookMark_HomePageTest" time="146.413" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.pom.crimson.listeners.LogAssert.onAssertFailure(LogAssert.java:41)
at org.testng.asserts.SoftAssert.doAssert(SoftAssert.java:23)
at org.testng.asserts.Assertion.fail(Assertion.java:132)
at com.pom.crimson.testcases.explore.smoke.HomePageTest.verifyBookMark_HomePageTest(HomePageTest.java:234)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyBookMark_HomePageTest -->
  <system-out/>
  <testcase name="verifyRemoveBookMark_HomePageTest" time="122.958" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyShareInGroupChat_HomePageTest" time="55.018" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
  <testcase name="verifyShareInOtherApps_HomePageTest" time="164.619" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Check if Messages app is displayed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Check if Messages app is displayed expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.crimson.testcases.explore.smoke.HomePageTest.verifyShareInOtherApps_HomePageTest(HomePageTest.java:406)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyShareInOtherApps_HomePageTest -->
  <system-out/>
  <testcase name="verifyNavigationToSavedArticlesPage_SavedArticlesPageTest" time="35.870" classname="com.pom.crimson.testcases.explore.smoke.HomePageTest"/>
  <system-out/>
</testsuite> <!-- com.pom.crimson.testcases.explore.smoke.HomePageTest -->
